nixos/tests/ejabberd.nix

```diff
@@ -0,0 +1,262 @@
+import ./make-test.nix ({ pkgs, ... }: {
```

Comment:
Please add this to all-tests.nix

Reply:
Thanks for walking me through this… I added it now.